src/utils.py

```diff
@@ -0,0 +1,108 @@
+# pylint: disable=missing-module-docstring
+import logging
+import os
+import sys
+from io import BytesIO
+from typing import Union
+
+import numpy as np
+from PIL import Image
+from tabulate import tabulate
+from viam.logging import getLogger
+from viam.media.video import CameraMimeType, ViamImage
+
+LOGGER = getLogger(__name__)
+SUPPORTED_IMAGE_TYPE = [
+    CameraMimeType.JPEG,
+    CameraMimeType.PNG,
+    CameraMimeType.VIAM_RGBA,
+]
+LIBRARY_SUPPORTED_FORMATS = ["JPEG", "PNG", "VIAM_RGBA"]
+
+
+def decode_image(image: Union[Image.Image, ViamImage]) -> np.ndarray:
```

Comment:
Definitely might be a thing already in Python SDK.  I actually don't care whether you use theirs or this but check out  viam/media/utils/pil in the Python SDK.

Reply:
I want to use orin GPU to decode in the future, should have mentioned it, but can I keep it like that for now?